# What was the impact of natural disasters in risk pool regions?

Data Source: 
EM-DAT: The Emergency Events Database - Université catholique de Louvain (UCL) - CRED, D. Guha-Sapir - https://www.emdat.be, Brussels, Belgium.

Downloaded on: 2018-11-22
    
FAQ for the database: https://www.emdat.be/frequently-asked-questions

Time frame: since risk pool establishment, so different time frames for each risk pool

Disasters that are included in this analysis: Out of all recorded natural disasters in the EMDAT database, this analysis looks at a subset of data which only includes natural disasters of the following categories: Riverine flood, Flash flood, Coastal flood, Tropical cyclone, Extra-tropical storm, Heat wave, Drought, Ground movement, Tsunami

Earthquakes (tsunamis and ground movements) were excluded later in the analysis

Damage cost is not available for all disasters recorded in the database, so value sums are MINIMUM damage (real ones higher)

Unit of damage cost: "The estimated damage is given in US $ (in thousands) in the value of the year of occurrence."

In [3]:
import pandas as pd
import matplotlib.pyplot as plt
%matplotlib inline

plt.rcParams['pdf.fonttype'] = 42
plt.rcParams['ps.fonttype'] = 42

In [2]:
df = pd.read_csv('data/EMDAT/EMDAT-climateinsurance-ev.csv', delimiter =';', encoding = 'latin-1')
df.head()

year  iso country_name continent                     region  \
0  1998  AUS    Australia   Oceania  Australia and New Zealand   
1  1998  JPN        Japan      Asia               Eastern Asia   
2  1998  MEX       Mexico  Americas            Central America   
3  1999  KHM     Cambodia      Asia         South-Eastern Asia   
4  1999  COL     Colombia  Americas              South America   

  disaster_subtype disaster_type disaster_subgroup  occurrence  Total_deaths  \
0    Coastal flood         Flood      Hydrological           2           4.0   
1    Coastal flood         Flood      Hydrological           1          16.0   
2    Coastal flood         Flood      Hydrological           1         274.0   
3    Coastal flood         Flood      Hydrological           1           NaN   
4    Coastal flood         Flood      Hydrological           1           3.0   

   Injured  Affected  Homeless  Total_affected  Total_damage_K_USD  
0      NaN    1375.0       NaN          1375.0            161300.0  
1     33.0   24000.0       NaN         24033.0                 NaN  
2      NaN   25000.0       NaN         25000.0            602700.0  
3      NaN  106670.0   17805.0        124475.0                 NaN  
4      NaN       NaN    1100.0          1100.0                 NaN

In [3]:
df['disaster_subtype'].value_counts()

Riverine flood          1191
Tropical cyclone         543
Flash flood              413
Drought                  347
Ground movement          318
Heat wave                132
Extra-tropical storm     105
Coastal flood             46
Tsunami                   30
Name: disaster_subtype, dtype: int64

In [1]:
# creating groups of countries that have received a payout (_recipients) 
# and that would have not received a payout (_potentials)

ccrif_recipients = ['Anguilla','Antigua & Barbuda','Bahamas','Barbados','Belize','Dominica','St. Kitts & Nevis','Saint Lucia','St. Vincent & the Grenadines','Trinidad & Tobago','Turks & Caicos Islands','Nicaragua','Haiti']
ccrif_potentials = ['Bermuda','Cayman Islands','Grenada','Jamaica']
ccrif_all = ccrif_recipients + ccrif_potentials


pcrafi_recipients = ['Tonga', 'Vanuatu']
pcrafi_potentials = ['Cook Islands','Marshall Islands','Samoa','Fiji','Kiribati','Micronesia','Nauru','Niue','Palau','Papua New Guinea','Solomon Islands','Tuvalu']
pcrafi_all = pcrafi_recipients + pcrafi_potentials


arc_recipients = ['Malawi','Mauritania','Niger','Senegal']
arc_potentials = ['Gambia','Kenya','Mali']
arc_optionals = ['Benin','Burkina Faso','Burundi','CAR','Chad','Comoros','Republic Congo','Djibouti','Gabon','Guinea','Guinea-Bissau','Ivory Coast','Liberia','Libya','Madagascar','Mozambique','Nigeria','Rwanda','Sahrawi','Sao Tome & Principe','Sierra Leone','Togo','Zambia','Zimbabwe']
arc_all = arc_recipients + arc_potentials


In [2]:
# make dfs that are policy for each risk pool as in regions covered and timepoint of establishment

ccrif_est = 2007
arc_est = 2014
pcrafi_est = 2016

df_ccrif = pd.DataFrame()
df_arc = pd.DataFrame()
df_pcrafi = pd.DataFrame()

pools = [ccrif_all, arc_all, pcrafi_all]

for risk_pool in pools:
    if 'Anguilla' in risk_pool:
        df_time_temp = df[df['year']>=ccrif_est]
        for country in risk_pool:
            df_temp = df_time_temp[df_time_temp['country_name']==country]
            df_ccrif = df_ccrif.append(df_temp)
    if 'Malawi' in risk_pool:
        df_time_temp = df[df['year']>=arc_est]
        for country in risk_pool:
            df_temp = df_time_temp[df_time_temp['country_name']==country]
            df_arc = df_arc.append(df_temp)
    if 'Tonga' in risk_pool:
        df_time_temp = df[df['year']>=pcrafi_est]
        for country in risk_pool:
            df_temp = df_time_temp[df_time_temp['country_name']==country]
            df_pcrafi = df_pcrafi.append(df_temp)

NameError: name 'pd' is not defined

In [6]:
risk_pool_dfs = [df_ccrif,df_arc,df_pcrafi]

In [7]:
df_ccrif.head()

year  iso country_name continent     region  disaster_subtype  \
3029  2017  AIA     Anguilla  Americas  Caribbean  Tropical cyclone   
253   2010  BRB     Barbados  Americas  Caribbean           Drought   
1061  2007  BRB     Barbados  Americas  Caribbean   Ground movement   
2876  2010  BRB     Barbados  Americas  Caribbean  Tropical cyclone   
3034  2017  BRB     Barbados  Americas  Caribbean  Tropical cyclone   

     disaster_type disaster_subgroup  occurrence  Total_deaths  Injured  \
3029         Storm    Meteorological           1           4.0      NaN   
253        Drought    Climatological           1           NaN      NaN   
1061    Earthquake       Geophysical           1           NaN      1.0   
2876         Storm    Meteorological           1           NaN      NaN   
3034         Storm    Meteorological           1           1.0      NaN   

      Affected  Homeless  Total_affected  Total_damage_K_USD  
3029   15000.0       NaN         15000.0            200000.0  
253        NaN       NaN             NaN                 NaN  
1061       NaN       NaN             1.0                 NaN  
2876    2500.0       NaN          2500.0                 NaN  
3034       NaN       NaN             NaN                 NaN

## how many people have been affected in the past by natural disasters in the risk pool regions

In [8]:
for rp in risk_pool_dfs:
    number_affected = rp['Affected'].sum()
    print("Since establishment of the risk pool", number_affected/1000000, "million people have been affected by a natural disaster there.")

Since establishment of the risk pool 13.624087 million people have been affected by a natural disaster there.
Since establishment of the risk pool 16.751402 million people have been affected by a natural disaster there.
Since establishment of the risk pool 1.279089 million people have been affected by a natural disaster there.


**Note:**
    
CCRIF: Affected 13 million, covered 12.5 million
**--> Less people covered than affected in since the establishment of the pool**
    
ARC: Affected 16 million, covered 150 million
    
PCRAFI: Affected 1 million, covered 4.5 million

## how many disasters happened in that time in that risk pool region

In [9]:
for rp in risk_pool_dfs:
    number_disasters = rp['occurrence'].sum()
    print("Since establishment of the risk pool", number_disasters , "disasters happened there.")

Since establishment of the risk pool 93 disasters happened there.
Since establishment of the risk pool 14 disasters happened there.
Since establishment of the risk pool 13 disasters happened there.


**Note:**

CCRIF: 93 natural disasters, 44 payouts (less than every second, only half of all disasters triggered a payout)

ARC: 14 natural disasters, 5 payouts (one in three disasters triggered a payout, a third of all disasters, 35% triggered a payout)

PCRAFI: 13 natural disasters, 3 payouts (one in five disasters, 23% triggered a payout)



## how much damage did disasters cause in the risk pool regions since establishment of the respective pool

In [10]:
for rp in risk_pool_dfs:
    damage_caused = rp['Total_damage_K_USD'].sum()
    print("Since establishment of the risk pool, natural disasters caused US$", damage_caused/1000 , "million of damage there.")

Since establishment of the risk pool, natural disasters caused US$ 13101.594 million of damage there.
Since establishment of the risk pool, natural disasters caused US$ 390.0 million of damage there.
Since establishment of the risk pool, natural disasters caused US$ 661.0 million of damage there.


**Note**:
    
CCRIF: US$ 13101 million (13 billion) damage caused, payouts covered 139.4 million (equals 1 percent) - payins by members: 226 million

ARC: US$ 390 million damage caused, payouts covered 36.6 million (equals 9.4 percent)

PCRAFI: US$ 661 million damage caused, payouts covered 6.7 million (equals 1 percent)

# only look at disasters insured under each risk pool

In [11]:
arc_disaster_types = ['Drought']
ccrif_disaster_types = ['Tropical cyclone', 'Riverine flood', 'Flash flood', 'Ground movement', 'Tsunami']
pcrafi_disaster_types = ['Tropical cyclone']

In [12]:
df_ccrif_policy = pd.DataFrame()
df_arc_policy = pd.DataFrame()
df_pcrafi_policy = pd.DataFrame()

pools = [ccrif_all, arc_all, pcrafi_all]

for risk_pool in pools:
    if 'Anguilla' in risk_pool:
        df_time_temp2 = df[df['year']>=ccrif_est]
        for country in risk_pool:
            df_temp2 = df_time_temp2[df_time_temp2['country_name']==country]
            for disaster_type in ccrif_disaster_types:
                df_type_temp = df_temp2[df_temp2['disaster_subtype']==disaster_type]
                df_ccrif_policy = df_ccrif_policy.append(df_type_temp)
    if 'Malawi' in risk_pool:
        df_time_temp2 = df[df['year']>=arc_est]
        for country in risk_pool:
            df_temp2 = df_time_temp2[df_time_temp2['country_name']==country]
            for disaster_type in arc_disaster_types:
                df_type_temp = df_temp2[df_temp2['disaster_subtype']==disaster_type]
                df_arc_policy = df_arc_policy.append(df_type_temp)
    if 'Tonga' in risk_pool:
        df_time_temp2 = df[df['year']>=pcrafi_est]
        for country in risk_pool:
            df_temp2 = df_time_temp2[df_time_temp2['country_name']==country]
            for disaster_type in pcrafi_disaster_types:
                df_type_temp = df_temp2[df_temp2['disaster_subtype']==disaster_type]
                df_pcrafi_policy = df_pcrafi_policy.append(df_type_temp)

In [13]:
df_arc_policy

year  iso country_name continent          region disaster_subtype  \
344  2015  MWI       Malawi    Africa  Eastern Africa          Drought   
379  2017  MRT   Mauritania    Africa  Western Africa          Drought   
327  2014  SEN      Senegal    Africa  Western Africa          Drought   
323  2014  KEN        Kenya    Africa  Eastern Africa          Drought   
364  2016  KEN        Kenya    Africa  Eastern Africa          Drought   

    disaster_type disaster_subgroup  occurrence  Total_deaths  Injured  \
344       Drought    Climatological           1           NaN      NaN   
379       Drought    Climatological           1           NaN      NaN   
327       Drought    Climatological           1           NaN      NaN   
323       Drought    Climatological           1           NaN      NaN   
364       Drought    Climatological           1           NaN      NaN   

      Affected  Homeless  Total_affected  Total_damage_K_USD  
344  6700000.0       NaN       6700000.0                 NaN  
379  3893774.0       NaN       3893774.0                 NaN  
327   639702.0       NaN        639702.0                 NaN  
323  1600000.0       NaN       1600000.0                 NaN  
364  3000000.0       NaN       3000000.0                 NaN

In [14]:
df_pcrafi_policy

year  iso country_name continent     region  disaster_subtype  \
3026  2016  TON        Tonga   Oceania  Polynesia  Tropical cyclone   
3091  2018  TON        Tonga   Oceania  Polynesia  Tropical cyclone   
3070  2017  VUT      Vanuatu   Oceania  Melanesia  Tropical cyclone   
3088  2018  WSM        Samoa   Oceania  Polynesia  Tropical cyclone   
3015  2016  FJI         Fiji   Oceania  Melanesia  Tropical cyclone   
3078  2018  FJI         Fiji   Oceania  Melanesia  Tropical cyclone   

     disaster_type disaster_subgroup  occurrence  Total_deaths  Injured  \
3026         Storm    Meteorological           3           NaN      NaN   
3091         Storm    Meteorological           1           NaN      NaN   
3070         Storm    Meteorological           1           NaN      NaN   
3088         Storm    Meteorological           1           NaN      NaN   
3015         Storm    Meteorological           2          47.0    144.0   
3078         Storm    Meteorological           1           5.0      NaN   

      Affected  Homeless  Total_affected  Total_damage_K_USD  
3026     392.0       NaN           392.0                 NaN  
3091   87000.0       NaN         87000.0                 NaN  
3070    2564.0       NaN          2564.0                 NaN  
3088       NaN       NaN             NaN                 NaN  
3015  545414.0       NaN        545558.0            600000.0  
3078   89950.0       NaN         89950.0                 NaN

In [15]:
df_ccrif_policy

year  iso country_name continent           region  disaster_subtype  \
3029  2017  AIA     Anguilla  Americas        Caribbean  Tropical cyclone   
2876  2010  BRB     Barbados  Americas        Caribbean  Tropical cyclone   
3034  2017  BRB     Barbados  Americas        Caribbean  Tropical cyclone   
1061  2007  BRB     Barbados  Americas        Caribbean   Ground movement   
2791  2007  BLZ       Belize  Americas  Central America  Tropical cyclone   
2829  2008  BLZ       Belize  Americas  Central America  Tropical cyclone   
2877  2010  BLZ       Belize  Americas  Central America  Tropical cyclone   
3012  2016  BLZ       Belize  Americas  Central America  Tropical cyclone   
1952  2008  BLZ       Belize  Americas  Central America    Riverine flood   
2794  2007  DMA     Dominica  Americas        Caribbean  Tropical cyclone   
2910  2011  DMA     Dominica  Americas        Caribbean  Tropical cyclone   
2991  2015  DMA     Dominica  Americas        Caribbean  Tropical cyclone   
3039  2017  DMA     Dominica  Americas        Caribbean  Tropical cyclone   
2821  2007  LCA  Saint Lucia  Americas        Caribbean  Tropical cyclone   
2899  2010  LCA  Saint Lucia  Americas        Caribbean  Tropical cyclone   
2388  2013  LCA  Saint Lucia  Americas        Caribbean    Riverine flood   
759   2010  LCA  Saint Lucia  Americas        Caribbean       Flash flood   
1072  2007  LCA  Saint Lucia  Americas        Caribbean   Ground movement   
2814  2007  NIC    Nicaragua  Americas  Central America  Tropical cyclone   
2845  2008  NIC    Nicaragua  Americas  Central America  Tropical cyclone   
2868  2009  NIC    Nicaragua  Americas  Central America  Tropical cyclone   
2895  2010  NIC    Nicaragua  Americas  Central America  Tropical cyclone   
3022  2016  NIC    Nicaragua  Americas  Central America  Tropical cyclone   
3057  2017  NIC    Nicaragua  Americas  Central America  Tropical cyclone   
1916  2007  NIC    Nicaragua  Americas  Central America    Riverine flood   
1996  2008  NIC    Nicaragua  Americas  Central America    Riverine flood   
2071  2009  NIC    Nicaragua  Americas  Central America    Riverine flood   
2156  2010  NIC    Nicaragua  Americas  Central America    Riverine flood   
2232  2011  NIC    Nicaragua  Americas  Central America    Riverine flood   
2301  2012  NIC    Nicaragua  Americas  Central America    Riverine flood   
2377  2013  NIC    Nicaragua  Americas  Central America    Riverine flood   
2433  2014  NIC    Nicaragua  Americas  Central America    Riverine flood   
2486  2015  NIC    Nicaragua  Americas  Central America    Riverine flood   
1170  2014  NIC    Nicaragua  Americas  Central America   Ground movement   
2800  2007  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
2837  2008  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
2885  2010  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
2913  2011  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
2935  2012  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
2993  2015  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
3044  2017  HTI        Haiti  Americas        Caribbean  Tropical cyclone   
1898  2007  HTI        Haiti  Americas        Caribbean    Riverine flood   
2053  2009  HTI        Haiti  Americas        Caribbean    Riverine flood   
2134  2010  HTI        Haiti  Americas        Caribbean    Riverine flood   
2214  2011  HTI        Haiti  Americas        Caribbean    Riverine flood   
2286  2012  HTI        Haiti  Americas        Caribbean    Riverine flood   
2356  2013  HTI        Haiti  Americas        Caribbean    Riverine flood   
2473  2015  HTI        Haiti  Americas        Caribbean    Riverine flood   
2511  2016  HTI        Haiti  Americas        Caribbean    Riverine flood   
734   2009  HTI        Haiti  Americas        Caribbean       Flash flood   
750   2010  HTI        Haiti  Americas        Caribbean       Flash flood   
80

In [16]:
ccrif_policy_damage = df_ccrif_policy['Total_damage_K_USD'].sum()
print('Insured events caused a damage of',ccrif_policy_damage/1000,'million USD.')

Insured events caused a damage of 11015.594 million USD.


Note: For CCRIF, even if you only include disaster types that are potentially covered by a risk insurance in the damage calculation, the damage caused is 11015 million (or 11 billion) US Dollar, compared to payouts of 131 million US Dollar (equals 1.2 percent)

In [17]:
ccrif_policy_damage_wo_eq = df_ccrif_policy[df_ccrif_policy['disaster_type']!= "Earthquake"]['Total_damage_K_USD'].sum()
print('Insured events caused a damage of',ccrif_policy_damage_wo_eq/1000,'million USD.')

Insured events caused a damage of 3012.594 million USD.


EXCLUDING EARTHQUAKES, for CCRIF, even if you only include disaster types that are potentially covered by a risk insurance in the damage calculation, the damage caused is 3012.594 million US Dollar, compared to payouts of 121.9 million US Dollar (equals 4 percent)

## Repeat first analysis, without earthquake data 

In [17]:
df_woe = df[df['disaster_type']!= 'Earthquake']
df_woe

year  iso                            country_name continent  \
0     1998  AUS                               Australia   Oceania   
1     1998  JPN                                   Japan      Asia   
2     1998  MEX                                  Mexico  Americas   
3     1999  KHM                                Cambodia      Asia   
4     1999  COL                                Colombia  Americas   
5     1999  MEX                                  Mexico  Americas   
6     1999  NIC                               Nicaragua  Americas   
7     1999  PAN                                  Panama  Americas   
8     1999  THA                                Thailand      Asia   
9     1999  VEN      Venezuela (Bolivarian Republic of)  Americas   
10    1999  VNM                                Viet Nam      Asia   
11    2000  BGD                              Bangladesh      Asia   
12    2000  FJI                                    Fiji   Oceania   
13    2000  JPN                                   Japan      Asia   
14    2000  PHL                       Philippines (the)      Asia   
15    2000  RUS                Russian Federation (the)    Europe   
16    2001  AUS                               Australia   Oceania   
17    2001  MWI                                  Malawi    Africa   
18    2001  MOZ                              Mozambique    Africa   
19    2001  RUS                Russian Federation (the)    Europe   
20    2001  VNM                                Viet Nam      Asia   
21    2002  CHN                                   China      Asia   
22    2002  HND                                Honduras  Americas   
23    2002  PHL                       Philippines (the)      Asia   
24    2003  DOM                Dominican Republic (the)  Americas   
25    2003  MWI                                  Malawi    Africa   
26    2003  MEX                                  Mexico  Americas   
27    2003  MOZ                              Mozambique    Africa   
28    2003  VNM                                Viet Nam      Asia   
29    2006  HTI                                   Haiti  Americas   
...    ...  ...                                     ...       ...   
3065  2017  LKA                               Sri Lanka      Asia   
3066  2017  TWN              Taiwan (Province of China)      Asia   
3067  2017  THA                                Thailand      Asia   
3068  2017  TCA          Turks and Caicos Islands (the)  Americas   
3069  2017  USA          United States of America (the)  Americas   
3070  2017  VUT                                 Vanuatu   Oceania   
3071  2017  VNM                                Viet Nam      Asia   
3072  2017  VGB                 Virgin Island (British)  Americas   
3073  2017  VIR                    Virgin Island (U.S.)  Americas   
3074  2017  ZWE                                Zimbabwe    Africa   
3075  2018  CHN                                   China      Asia   
3076  2018  CUB                                    Cuba  Americas   
3077  2018  DJI                                Djibouti    Africa   
3078  2018  FJI                                    Fiji   Oceania   
3079  2018  HKG                               Hong Kong      Asia   
3080  2018  IND                                   India      Asia   
3081  2018  JPN                                   Japan      Asia   
3082  2018  LAO  Lao People's Democratic Republic (the)      Asia   
3083  2018  MDG                              Madagascar    Africa   
3084  2018  MUS                               Mauritius    Africa   
3085  2018  OMN                                    Oman      Asia   
3086  2018  PHL                       Philippines (the)      Asia   
3087  2018  REU                                 Réunion    Africa   
3088  2018  WSM                                   Samoa   Oceania   
3089  2018  SOM                                 Somalia    Africa   
3090  2018  TWN              Taiwan (Province of China)      Asia   
3091

In [18]:
df_ccrif_woe = pd.DataFrame()
df_arc_woe = pd.DataFrame()
df_pcrafi_woe = pd.DataFrame()

for risk_pool in pools:
    if 'Anguilla' in risk_pool:
        df_time_temp3 = df_woe[df_woe['year']>=ccrif_est]
        for country in risk_pool:
            df_temp3 = df_time_temp3[df_time_temp3['country_name']==country]
            df_ccrif_woe = df_ccrif_woe.append(df_temp3)
    if 'Malawi' in risk_pool:
        df_time_temp3 = df_woe[df_woe['year']>=arc_est]
        for country in risk_pool:
            df_temp3 = df_time_temp3[df_time_temp3['country_name']==country]
            df_arc_woe = df_arc_woe.append(df_temp3)
    if 'Tonga' in risk_pool:
        df_time_temp3 = df_woe[df_woe['year']>=pcrafi_est]
        for country in risk_pool:
            df_temp3 = df_time_temp3[df_time_temp3['country_name']==country]
            df_pcrafi_woe = df_pcrafi_woe.append(df_temp3)

In [19]:
risk_pool_dfs_woe = [df_ccrif_woe,df_arc_woe,df_pcrafi_woe]

for rp in risk_pool_dfs_woe:
    number_affected_woe = rp['Affected'].sum()
    print("Excluding earthquakes, since establishment of the risk pool", number_affected_woe/1000000, "million people have been affected by a natural disaster there.")

Excluding earthquakes, since establishment of the risk pool 9.862072 million people have been affected by a natural disaster there.
Excluding earthquakes, since establishment of the risk pool 16.751402 million people have been affected by a natural disaster there.
Excluding earthquakes, since establishment of the risk pool 0.72532 million people have been affected by a natural disaster there.


In [20]:
for rp in risk_pool_dfs_woe:
    number_disasters_woe = rp['occurrence'].sum()
    print("Excluding earthquakes, since establishment of the risk pool", number_disasters_woe , "disasters happened there.")

Excluding earthquakes, since establishment of the risk pool 88 disasters happened there.
Excluding earthquakes, since establishment of the risk pool 14 disasters happened there.
Excluding earthquakes, since establishment of the risk pool 9 disasters happened there.


CCRIF: 88 natural disasters, 38 payouts (less than every second, less than half of all disasters triggered a payout)

ARC: 14 natural disasters, 5 payouts (one in three disasters triggered a payout, a third of all disasters, 35% triggered a payout)

PCRAFI: 9 natural disasters, 3 payouts (one in three disasters, 33% triggered a payout)

In [21]:
for rp in risk_pool_dfs_woe:
    damage_caused_woe = rp['Total_damage_K_USD'].sum()
    print("Excluding earthquakes, since establishment natural disasters caused US$", damage_caused_woe/1000 , "million of damage there.")

Excluding earthquakes, since establishment natural disasters caused US$ 5098.594 million of damage there.
Excluding earthquakes, since establishment natural disasters caused US$ 390.0 million of damage there.
Excluding earthquakes, since establishment natural disasters caused US$ 600.0 million of damage there.


**Note**:
    
CCRIF: US$ 5098 million (5 billion) damage caused, payouts covered 121.9 million (equals 2.4 percent) - payins by members: 226 million

ARC: US$ 390 million damage caused, payouts covered 36.6 million (equals 9.4 percent)

PCRAFI: US$ 600 million damage caused, payouts covered 6.7 million (equals 1.1 percent)